demisto_sdk/commands/validate/default_config.toml

```diff
@@ -1,79 +1,34 @@
 ignorable_errors = [
-    "BA101", "BA106", "BA108", "BA109", "BA110", "BA111", "BA112", "BA113", "BA116", "BA119", "BA124", "BA125",
-    "GF102",
-    "DS107", "DS108",
-    "IF100", "IF106", "IF113", "IF115", "IF116",
-    "IN107", "IN109", "IN110", "IN122", "IN123", "IN124", "IN126", "IN128", "IN135", "IN136", "IN139", "IN144", "IN145", "IN150", "IN153", "IN154", "IN161",
-    "MP106",
-    "PA113", "PA116", "PA124", "PA125", "PA127", "PA129",
-    "PB104", "PB105", "PB106", "PB107", "PB110", "PB111", "PB114", "PB115", "PB116", "PB118", "PB119", "PB121",
-    "RM100", "RM102", "RM104", "RM106", "RM108", "RM110", "RM112", "RM113",
-    "RP102", "RP104",
-    "SC100", "SC105", "SC106",
-    "IM111",
-    "RN112", "RN113", "RN114", "RN115", "RN116",
-    "MR108",
-    "PR101",
-    "LO107",
-    "DB100",
-    "GR103",
-    "SC105",
-    "RM113"
+    "BA101", "BA105", "BA106", "BA116", "BA118",
```

Comment:
we should have a section called xsoar_reccomended that lets customers run the same validations they have been using until today
Lets also go over this file in our pre-release check-in with @anara123

Reply:
You mean with all the validations we have atm?
Shouldn't it be up for the customer to configure?

Reply:
Will add a section of sections that mock the flows that run in our builds under 'xsoar_best_practices' section.